### ISSUE TITLE
Update illustrations to include cell type report

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-docs#169]
We will need to update all of our illustrations on the download page to account for the new cell type report that will be included in downloads. 

The new structure is as follows: 

Project Download
```
project_id
├── README.md
├── bulk_metadata.tsv
├── bulk_quant.tsv
├── sample_id
│   ├── SCPCL000000_filtered.rds
│   ├── SCPCL000000_processed.rds
│   ├── SCPCL000000_qc_report.html
|   ├── SCPCL000000_celltype_report.html*
│   ├── SCPCL000000_unfiltered.rds
└── single_cell_metadata.tsv
```

Sample Download
```
sample_id
├── README.md
├── SCPCL000000_filtered.rds
├── SCPCL000000_processed.rds
├── SCPCL000000_qc_report.html
├── SCPCL000000_celltype_report.html*
├── SCPCL000000_unfiltered.rds
└── single_cell_metadata.tsv
```

Note that we will need to make the updates in both illustrations for downloading SCE objects and AnnData objects. For all illustrations, the new report should be added at the same level as the RDS/HDF5 files (within the sample_id folder).

**Comment by dvenprasad** [comment]
Note to self: Add accessed data to the project-id and sample-id

**Comment by dvenprasad** [comment]
**Single-cell Project Download**
![sc-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/6ac836cd-2198-4f38-abd4-993983d00015)

**Single-cell Sample Download**
![sc-sample-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/2ee9ce5c-082c-48da-ba40-2007be48b7f1)

**AnnData Project Download**
![anndata-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/5b21dec5-50f8-4fa3-b67f-96dbbd555857)

**AnnData Sample Download**
![anndata-sample-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/35432811-6d37-44b1-83ea-3b7f9e2a272b)

**AnnData CITE-seq Download**
![anndata-CITE-seq-sample-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/5002b97e-825d-4b7b-9583-48938cf4f386)

**Comment by dvenprasad** [comment]
Changing the `<download_date>` to `<accessed-date>` per #176 and sizing up to improve image quality

**Single-cell Project Download**
![sc-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/bca2d394-abc3-4b72-802c-c4adedbdfb6e)

**Single-cell Sample Download**
![sc-sample-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/6cb79cbd-269d-4f80-a388-da7e312ca555)

**AnnData Project Download**
![anndata-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/0817db2b-3ace-40ea-ba04-939165643729)

**AnnData Sample Download**
![anndata-sample-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/6abdf383-8157-4726-b09a-dcc1de647af3)

**AnnData CITE-seq Download**
![anndata-CITE-seq-sample-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/58b34e1f-9fef-4a29-b39e-01d9fa7015f9)

**Comment by dvenprasad** [comment]
Does this look correct? 

![multiplexed-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/02f573d9-bbfc-4947-bc9d-ae63db9eb4c5)

**Comment by allyhawkins** [comment]
Perfect, thank you!

**Comment by allyhawkins** [comment]
Closed by #191
